hw0603uitask2/src/main/java/com/tashad16a/gmail/hw0603ui/uigooglepodcasts/ViewPagerAdapter.java

```diff
@@ -0,0 +1,47 @@
+package com.tashad16a.gmail.hw0603ui.uigooglepodcasts;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import com.tashad16a.gmail.hw0603_ui.R;
+
+public class ViewPagerAdapter extends FragmentPagerAdapter {
+    private Context mContext;
+
+    public ViewPagerAdapter(Context context, FragmentManager fragmentManager) {
+        super(fragmentManager);
+
+        mContext = context;
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        if (position == 0) {
+            return new PageFragment();
```

Comment:
in three cases you return same fragment.
Also better reuse exists one, that create new one when getItem called

Reply:
```suggestion
           @Override
    public Fragment getItem(int position) {
        PageFragment pageFragment = new PageFragment();
        SimplePageFragment simplePageFragment = new SimplePageFragment();

        if (position == 0) {
            return pageFragment;
        } else {
            //TODO change text use CompoundView...  updateTextFragment(getPageTitle(position))
            return simplePageFragment;
        }
    }
```